### PR TITLE
Add de.philippun1.Snoop

### PR DESCRIPTION
**Pull request by philippun1**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by philippun1** [comment]
bot, build de.philippun1.Snoop

File: de.philippun1.Snoop.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    ],
+    "modules" : [
+        {
+            "name" : "omgrep",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources": [
+              {
+                "type": "archive",
+                "url": "https://gitlab.gnome.org/philippun1/omgrep/-/archive/0.1/omgrep-0.1.tar.gz",
```
Comment (hfiguiere) [inline de.philippun1.Snoop.json:35]: onmgrep but the app name is "Snoop"?

Comment (philippun1) [reply de.philippun1.Snoop.json:35]: Snoop is the friendly name, omgrep the repo name (and original name). Seems to give some problems with the appdata file.
Is there a way to make it work? Or should I do a proper rename?

Comment (hfiguiere) [reply de.philippun1.Snoop.json:35]: the name of the files are important. so yeah.

**Comment by philippun1** [comment]
bot, build de.philippun1.Snoop

**Comment by philippun1** [comment]
@hfiguiere I renamed everything and it works now.

**Pull request closed without merging**